rocAL/rocAL/include/pipeline/node.h

```diff
@@ -54,5 +55,6 @@ class Node
     vx_node _node = nullptr;
     size_t _batch_size;
     pMetaDataBatch _meta_data_info;
-    vx_array _src_roi_width, _src_roi_height; // TODO - To be removed after tensor changes in augmentation nodes.
+    vx_scalar _input_layout, _output_layout, _roi_type;
```

Comment:
node accepts a vector of inputs and outputs. Can't we get the layout from the Tensor object pointer? Please change this

Reply:
All our nodes currently process 1 input and 1 output
All the OpenVX RPP nodes also accept only 3 params i.e input_layout, output_layout and roiType

Hence we have added this as part of the base node class so we need not replicate in every derived nodes.